src/components/ContentHeader.vue

```diff
@@ -1,72 +1,62 @@
 <template>
   <div>
-    <div class="usa-overlay"></div>
 <header class="usa-header usa-header--basic">
   <div class="usa-nav-container">
-    <div class="usa-navbar">
-      <div class="usa-logo" id="basic-logo">
-        <em class="usa-logo__text">
-          <a href="javascript:void(0)" title="USGS Vizlab">
-            USGS Vizlab
-          </a>
-        </em>
-      </div>
-      <button class="usa-menu-btn">Menu</button>
-    </div>
     <nav aria-label="Primary navigation" class="usa-nav">
-      <button class="usa-nav__close">
-        <img src="/assets/img/usa-icons/close.svg" role="img" alt="Close" />
-      </button>
       <ul class="usa-nav__primary usa-accordion">
+         <li class="usa-nav__primary-item">
+          <button
+            class="usa-accordion__button usa-nav__link"
+            aria-expanded="false"
+            aria-controls="basic-nav-new"
+            onClick="document.getElementById('viz-new').scrollIntoView();"
+          >
+            <span>What's new</span>
+          </button>
+        </li>
         <li class="usa-nav__primary-item">
           <button
-            class="usa-accordion__button usa-nav__link usa-current"
+            class="usa-nav__button usa-nav__link usa-current"
             aria-expanded="false"
-            aria-controls="basic-nav-section-one"
+            aria-controls="basic-nav-viz"
+            onClick="document.getElementById('viz-cards').scrollIntoView();"
           >
             <span>Visualizations</span>
           </button>
 
-          <ul id="basic-nav-section-one" class="usa-nav__submenu">
+    <!--       <ul id="basic-nav-section-one" class="usa-nav__submenu">
             <li class="usa-nav__submenu-item">
               <a href="#">Interactives</a>
             </li>
             <li class="usa-nav__submenu-item">
-              <a href="#">Hurricanes</a>
+              <a href="#">Chart Gallery</a>
             </li>
               <li class="usa-nav__submenu-item">
-              <a href="#">U.S. River Conditions</a>
-            </li>
-            <li class="usa-nav__submenu-item">
-              <a href="#">Chart Gallery</a>
+              <a href="#">Hurricanes</a>
             </li>
-          </ul>
+          </ul> -->
         </li>
         <li class="usa-nav__primary-item">
           <button
-            class="usa-accordion__button usa-nav__link"
+            class="usa-nav__button usa-nav__link"
             aria-expanded="false"
-            aria-controls="basic-nav-section-two"
+            aria-controls="basic-nav-about"
+            onClick="document.getElementById('viz-about').scrollIntoView();"
```

Comment:
Tried setting this to the div, too --  `onClick="document.getElementById('about-container').scrollIntoView();"`, but that didn't work. Not sure why?